apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -832,6 +836,28 @@ type DebugConfig struct {
 	Env map[string]string `json:"env,omitempty"`
 }
 
+// `SubnetLabels` allows to define custom labels on subnets and IPs or to enable automatic labelling of recognized subnets in OpenShift.
+type SubnetLabels struct {
+	// `openShiftAutoDetect` allows, when set to `true`, to detect automatically the machines, pods and services subnets based on the
+	// OpenShift install configuration and the Cluster Network Operator configuration.
+	//+optional
+	OpenShiftAutoDetect *bool `json:"openShiftAutoDetect,omitempty"`
+
+	// `customLabels` allows to customize subnets and IPs labelling, such as to identify cluster-external workloads or web services.
```

Comment:
We should mention that this can be used on top of OpenshiftAutoDetect option, overriding values in case of duplicates ?